base/set.jl

```diff
@@ -1,5 +1,33 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+	Set{T} <: AbstractSet{T}
+
+The default implementation of hash sets.
+
+`Set`s are containers that provide fast lookup and membership testing.
+Elements in a `Set` are unique, as determined by the elements' implementation of
+`isequal` and `hash`.
+The order of elements in a `Set` is an implementation detail and cannot be relied on.
+
+See also: [`AbstractSet`](@ref), [`BitSet`](@ref), [`Dict`](@ref)
+
+# Examples
+```jldoctest filter = r"^Set{[A-Za-z0-9]+} with [0-9]+ element(s)?:"
+julia> Set()
+Set{Any}()
+
+julia> Set(1)
+Set{Int64} with 1 element:
+  1
+
+julia> Set(['a', 'b', 'c'])
```

Comment:
```suggestion
julia> Set(['a', 'b', 'c', 'a'])
```

to show the uniqueness?